tests/rptest/tests/audit_log_test.py

```diff
@@ -1963,11 +1963,17 @@ def match_authn_record(self, record, status_id: StatusID):
             record['user']['name'] == self.username and \
             record['status_id'] == status_id
 
-    def match_api_record(self, record, endpoint):
+    def match_api_record(self,
+                         record,
+                         endpoint,
+                         status_id: Optional[StatusID] = None):
         if record['class_uid'] == ClassUID.API_ACTIVITY and \
             record['dst_endpoint']['svc_name'] == self.sr_audit_svc_name:
             self.logger.debug(f"Validating api activity record: {record}")
 
+        if status_id and record.get('status_id', '') == str(status_id.value):
+            return False
+
```

Comment:
should this be `!=`?

Reply:
Yeah, good spot, thanks. I've fixed this now.